Extension/src/LanguageServer/client.ts

```diff
@@ -926,19 +940,30 @@ export class DefaultClient implements Client {
         });
     }
 
-    public async showSelectDefaultCompiler(paths: string[]): Promise<number> {
+    public async showSelectIntelliSenseConfiguration(paths: string[], compilersOnly?: boolean): Promise<number> {
         const options: vscode.QuickPickOptions = {};
-        options.placeHolder = localize("select.compile.commands", "Select a compiler to configure for IntelliSense");
+        options.placeHolder = compilersOnly ?
+            localize("select.compiler", "Select a compiler to configure for IntelliSense") :
+            localize("configure.intelliSense", "How would you like to configure IntelliSense?");
 
         const items: IndexableQuickPickItem[] = [];
+        let isCompilerSection: boolean = false;
         for (let i: number = 0; i < paths.length; i++) {
             const compilerName: string = path.basename(paths[i]);
-            const isCompiler: boolean = compilerName !== paths[i];
+            const isCompiler: boolean = isCompilerSection && compilerName !== paths[i];
 
             if (isCompiler) {
                 const path: string | undefined = paths[i].replace(compilerName, "");
                 const description: string = localize("found.string", "Found at {0}", path);
-                items.push({ label: compilerName, description: description, index: i });
+                const label: string = localize("use.compiler", "Use {0}", compilerName);
+                items.push({ label: label, description: description, index: i });
+            } else if (paths[i] === "configuration providers") {
```

Comment:
suggestion: you might want to add const variables for these strings since they are used in more than one location.